exercises/sgf-parsing/test/Tests.hs

```diff
@@ -1,89 +1,39 @@
 {-# LANGUAGE OverloadedStrings #-}
 
-import Test.HUnit (Assertion, (@=?), runTestTT, Test(..), Counts(..))
-import System.Exit (ExitCode(..), exitWith)
-import Data.Map (Map)
-import qualified Data.Map as Map
-import Data.Tree
-import Data.Text (Text)
-import Sgf (parseSgf)
-
--- The Sgf module should export a parseSgf module with the following
--- signature:
---
--- parseSgf :: Text -> Maybe (Tree (Map Text [Text]))
---
--- You may find it useful to copy the type definitions for SgfTree and
--- SgfNode from this file.
---
--- The parsec library is part of the Haskell Platform. Please use it to
--- your advantage.
-
--- | A tree of nodes.
-type SgfTree = Tree SgfNode
-
--- | A node is a property list, each key can only occur once.
---
--- Keys may have multiple values associated with them.
-type SgfNode = Map Text [Text]
-
-exitProperly :: IO Counts -> IO ()
-exitProperly m = do
-  counts <- m
-  exitWith $ if failures counts /= 0 || errors counts /= 0 then ExitFailure 1 else ExitSuccess
-
-testCase :: String -> Assertion -> Test
-testCase label assertion = TestLabel label (TestCase assertion)
+import Data.Foldable     (for_)
+import Data.Map          (fromList)
+import Data.Tree         (Tree(Node))
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
 
--- | Feed the input to 'parseSgf' and check that the result matches what's
---   expected.
-tps :: Text -> Maybe SgfTree -> Test
-tps input expected =
-    let label = "parse " ++ show input
-    in testCase label (expected @=? parseSgf input)
+import Sgf (parseSgf)
 
 main :: IO ()
-main = exitProperly $ runTestTT $ TestList
-       [ TestList sgfParsingTests ]
-
-t :: [(Text, [Text])] -> [SgfTree] -> SgfTree
-t n = Node (Map.fromList n)
```

Comment:
Ah good, I think it was good to remove these, if I had to keep looking back on them to figure out how the test worked it would hvae made things more confusing.